src/main/groovy/life/qbic/samplestatus/reporter/services/QbicSampleTrackingService.groovy

```diff
@@ -33,54 +30,36 @@ class QbicSampleTrackingService implements SampleTrackingService {
     @Value('${service.sampletracking.url}')
     private String sampleTrackingBaseUrl
 
-    @Value('${service.sampletracking.location.endpoint}')
-    private String locationEndpoint
+    @Value('${service.sampletracking.endpoint}')
+    private String endpoint
 
     @Value('${service.sampletracking.auth.user}')
     private String serviceUser
 
     @Value('${service.sampletracking.auth.password}')
     private String servicePassword
 
-    private String locationEndpointPath
+    private String endpointPath
 
     @PostConstruct
     void initService() {
-        locationEndpointPath = sampleTrackingBaseUrl + locationEndpoint
+        endpointPath = sampleTrackingBaseUrl + endpoint
     }
 
-    @Override
-    Optional<Location> getLocationForUser(String userId) {
-        URI requestURI = createUserLocationURI(userId)
-        HttpResponse response = requestLocation(requestURI)
-        return Optional.of(response.body()).flatMap(DtoMapper::parseLocationOfJson)
-    }
-
-    private HttpResponse<String> requestSampleUpdate(URI requestURI, String locationJson) {
+    private HttpResponse<String> requestSampleUpdate(URI requestURI, String statusJson) {
         HttpRequest request = HttpRequest
                 .newBuilder(requestURI)
                 .header("Content-Type", "application/json")
-                .PUT(HttpRequest.BodyPublishers.ofString(locationJson))
+                .PUT(HttpRequest.BodyPublishers.ofString(statusJson))
```

Comment:
do we need to request a sample update? I guess before this was to verify if a valid location makes the update?